.travis.yml

```diff
@@ -9,13 +9,16 @@ before_install:
   - npm i -g npm@^3.0.0
   - npm i -g surge
   - npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN
-after_success:
+install:
+  - npm i
```

Comment:
I had to replace default "install" to add the "run-all" step. But I'll remove it and add run-all to the postinstall. Makes sense.